docs/release-notes/8.3.asciidoc

```diff
@@ -1,10 +1,27 @@
 [[release-notes-header-8.3.0]]
 == 8.3
 
+[discrete]
+[[release-notes-8.3.3]]
+=== 8.3.3
+
+[discrete]
+[[bug-fixes-8.3.3]]
+==== Bug fixes and enhancements
+* Fixes a bug that prevented the *Create field* button from appearing in the Fields browser when you accessed it from a Timeline created using the Alerts page's *Open in timeline* button ({pull}135842[#135842]).
+* Removes the unsupported `matches` operator from the *Add Rule Exception* flyout ({pull}136340[#136340]).
+* Prevents Rule Execution Log events from being wrongly ordered when max events are hit and events are filtered by status ({pull}131675[#131675]).
```

Comment:
If I missed a reason to cap this LMK! Also double check my revision to ensure it's accurate please :)
```suggestion
* Prevents rule execution log events from being wrongly ordered when the maximum number of events are reached and events are filtered by status ({pull}131675[#131675]).
```

Reply:
@banderror @vitaliidm would one of you mind verifying that this revision is correct from a technical standpoint? Thank you!

Reply:
@nastasha-solomon we used to call it `Rule Execution Log` in our tickets because it refers to a certain area in the codebase that serves as a foundation for a number of features in the app, so it's basically a title for us. Lowercase here LGTM 👍